openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
So I'm trying to avoid using the underlying almighty-core/log package as that brings in goa which then borks on jwt dependency clashes with kubernetes. It seems to work fine without that `InitTenant` controller callback & I could just replica whatever it does if I need to.